app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterItem.jsx

```diff
@@ -31,19 +33,42 @@ const stopPropagation = (func) => (e) => {
   func(e);
 };
 
+const handleClick = (e) => {
+  e.preventDefault();
+};
+
+const getLaunchesPageLink = (filter, projectId, allLatest, active) => {
```

Comment:
What about to transform this function to selector? Also I think it would be better to connect it directly to FilterItemBase and don't pass redundant props through whole hierarchy.

Reply:
@AmsterGet What do you think about transformation to the selector?

Reply:
@KatsiarynaTatarynovich please see the comment https://github.com/reportportal/service-ui/pull/2337#discussion_r393594676